Automattic/jetpack
# VideoPress: enqueue token bridge file in the front-end only when required

This PR iterates over how the app enqueues the token-bridge file. Currently, it's always enqueued in the front end.

* First, it removes the code line when enqueued in the front end.
* And it keeps relying only on hooking to the embed_oembed_html filter.

Fixes https://github.com/Automattic/jetpack/issues/24765

## Proposed changes:

* VideoPress: enqueue token bridge file in the front-end only when required

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Create a post without a VideoPress video block
* Save the post
* Go to the front end
* **Before**, confirm the client loads the token-bridge file

<img width="1193" alt="Screen Shot 2023-04-18 at 10 56 35" src="https://user-images.githubusercontent.com/77539/232800301-643d7d7f-2a34-4d9c-b017-05a1b5695111.png">

*  **After**,  confirm the client does not load the token-bridge file.

* Go back to the editor
* Create a VideoPress video block
* Set video privacy private 
* Go to the front end again
* Confirm the token-bridge file is now loaded
* You can confirm the bridge provides the token to the client by using debug() lines

```
localStorage.setItem( 'debug', 'videopress:*' )
```
